api/resources_portal/views/organization.py

```diff
@@ -0,0 +1,70 @@
+from rest_framework import serializers, viewsets
+
+from resources_portal.models import Organization, User
+from resources_portal.views.user import UserRelationSerializer
+
+
+class OrganizationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Organization
+        fields = (
+            "id",
+            "owner",
+            "members",
+            "created_at",
+            "updated_at",
+        )
+        read_only_fields = ("id", "created_at", "updated_at")
+
+
+class OrganizationDetailSerializer(OrganizationSerializer):
+    owner = UserRelationSerializer()
+    members = UserRelationSerializer(many=True)
+
+    def update_members(self, organization, members):
+        if not members:
+            # Nothing to do.
+            return organization
+
+        member_ids = [member["id"] for member in members]
+
+        organization.members.set(User.objects.filter(id__in=member_ids))
+        organization.save()
```

Comment:
should we return the updated model since we are silently not adding invalid members?